Vikta-UI-testing-sample/src/test/java/com/griddynamics/qa/vikta/uitesting/sample/stepsDefinitions/AddressSteps.java

```diff
@@ -0,0 +1,181 @@
+package com.griddynamics.qa.vikta.uitesting.sample.stepsDefinitions;
+
+import com.griddynamics.qa.vikta.uitesting.sample.pageObjects.AddressEditAddPage;
+import com.griddynamics.qa.vikta.uitesting.sample.pageObjects.AddressesListPage;
+import com.griddynamics.qa.vikta.uitesting.sample.pageObjects.BasePage;
+import com.griddynamics.qa.vikta.uitesting.sample.pageObjects.HomePage;
+import com.griddynamics.qa.vikta.uitesting.sample.utils.Utilities;
+import io.qameta.allure.Step;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import java.util.List;
+import java.util.StringJoiner;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class AddressSteps extends BaseSteps{
+
+    public AddressSteps(WebDriver driver) {
+        super(driver);
+    }
+
+    public enum AddressField {
+        STREET,
+        ADDITIONAL_STREET_INFO,
+        CITY,
+        REGION,
+        POSTAL_CODE,
+        NICKNAME
+    }
+
+    @Step
+    public String typeValueInto(AddressField addressField){
+        Utilities utilities = new Utilities();
```

Comment:
Utilites classes usually  contain only static methods so that  we don't  have to create an instance of them. Please refactor. You can also make  private constructor so no one can instantiate it